### ISSUE TITLE
Support for intent alternatives?

### ISSUE THREAD

**Issue by koenvervloesem** [rhasspy/rhasspy-hermes#20]
Rhasspy Hermes doesn't seem to support [intent alternatives](https://docs.snips.ai/reference/dialogue#intent-alternative) in the `hermes/intent/<INTENT>` messages. This is supposed to return alternatives for a detected intent, if you not only want to see the number one detected intent but a top three or something like that.

Is this something Rhasspy could support and should be put on the TODO list?

**Comment by synesthesiam** [comment]
Yes, this should be supported in the future. It will work differently across the NLU services, but the same basic idea holds.
